src/resources/js/events-admin.js

```diff
@@ -373,21 +373,26 @@ jQuery( document ).ready( function( $ ) {
 
 	var toggle_linked_post_fields = function( event ) {
 
-		var $select    = $( this );
-		var selectData = $select.data( 'select2' );
-		var $group     = $select.closest( 'tbody' );
-		var $edit      = $group.find( '.edit-linked-post-link a' );
-		var value      = $select.val();
-		var editLink   = '';
-
-		if ( $select.find( ':selected' ).val() == value ) {
-			editLink = $select.find( ':selected' ).data( 'editLink' );
+		const $select = $( this );
+		const $group = $select.closest( 'tbody' );
+		const $edit = $group.find( '.edit-linked-post-link a' );
+		const value = $select.val();
+		const $selected = $select.find( ':selected' );
+		const selectedVal = $selected.val();
+		let editLink = '';
+		let existingPost = false;
+
+		if ( selectedVal === value ) {
+			editLink = $selected.data( 'editLink' );
+			existingPost = !! $selected.data( 'existingPost' );
```

Comment:
I'm good with adjusting this to remove the `!!` but I wanted to make sure it enforced a boolean even if it was undefined or an integer.

Reply:
I like, @paulmskim do you have anything on this?

Reply:
What are the possible values for `data-existing-post`? Is it always available? I would keep the `!!`, probably the best way other than using `Boolean()`.

Reply:
Wow I did a smart thing!

Also the existing post value is output via PHP in this PR as "1" so it's only ever going to be "1" or undefined.

Reply:
Then this is fine. If you also had `'0'` as an option and you meant that to be false then `!!` would not work. But in this case this is fine.